br/backup-and-restore-design.md

```diff
@@ -0,0 +1,47 @@
+---
+title: BR Design Principles
+summary: Learn about the design details of BR.
+---
+
+# BR Design Principles
+
+This document describes the design principles of BR, including its architecture<!--, design philosophy,//--> and backup files.
+
+## BR architecture
+
+BR sends a backup or restoration command to each TiKV node. After receiving the command, TiKV performs the corresponding backup or restoration operation.
+
+Each TiKV node has a path in which the backup files generated in the backup operation are stored and from which the stored backup files are read during the restoration.
```

Comment:
```suggestion
Each TiKV node will access the storage path during backup and  restoration.
```

Reply:
Agree with this modification. Maybe we should synchronize the Chinese as well.